docs/read_next.rst

```diff
@@ -1,3 +1,29 @@
-What to read next
+What to Read Next
 =================
 
+Depending on your role, certain aspects of CumulusCI may pertain to you more than others.
+Use this section as a guide for which parts of the documentation are most likely to be relevant to you.
+
+For everyone
```

Comment:
I really like this section, as well as its placement at the end of a brief and cogent intro. We will need to take care that it doesn't get too bloated and increase risk of overwhelming first-timers.